app/components/variable-height-textarea.js

```diff
@@ -3,6 +3,7 @@ import Ember from 'ember';
 export default Ember.TextArea.extend({
   tagName: "textarea",
   attributeBindings: ["disabled"],
+  classNames: 'message_textbar',
```

Comment:
Follow hyphenated words for class name

Reply:
sure @Darshan-Chauhan , i'll use hyphenated class name.